app/components/WebsocketProvider.tsx

```diff
@@ -254,9 +254,7 @@ class WebsocketProvider extends React.Component<Props> {
     });
 
     this.socket.on("comments.delete", (event: WebsocketEntityDeletedEvent) => {
-      comments.inThread(event.modelId).forEach((comment) => {
```

Comment:
This is the type of logic no longer needed once the relationships are defined on the model.